### PR TITLE
Update benchmark  for v1-improvement

### PR DESCRIPTION
**Pull request by sifnoc**
Related to issue #197:

- We will now use the CSV file naming convention: "[N_CURRENCIES]__entry_2_[LEVELS]".
- Updated benchmark results based on PR #217.

### CODE REVIEW

**Review by enricobottazzi — CHANGES_REQUESTED** [review]

File: zk_prover/test.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (sifnoc) [reply zk_prover/test.sh]: It's leftover, removed at https://github.com/summa-dev/summa-solvency/pull/220/commits/298e83efc239e4dcd45c7b0794178b44695d8e5b

File: zk_prover/README.md [hunk 2]
```diff
@@ -84,15 +84,14 @@ The benchmarking included the following areas:
...
 In order to run the benchmarking, we provide a set of dummy `username, balances` entries formatted in csv files. The csv files can be downloaded as follows
 
-```
-cd benches
-mkdir csv
-cd csv
-wget https://summa-solvency.s3.eu-central-1.amazonaws.com/csv_files.tar.bz2
-tar -xjf csv_files.tar.bz2
+```bash
+mkdir -p benche/csv
```
[the remark this answers is not in the document]
Comment (sifnoc) [reply zk_prover/README.md]: Fixed at https://github.com/summa-dev/summa-solvency/pull/220/commits/298e83efc239e4dcd45c7b0794178b44695d8e5b

File: zk_prover/README.md [hunk 3]
```diff
@@ -84,15 +84,14 @@ The benchmarking included the following areas:
...
+cd benche/csv
+wget https://summa-solvency.s3.eu-central-1.amazonaws.com/1_entry_2_20.zip
+unzip sample_csv_files_for_bench.zip
```
[the remark this answers is not in the document]
Comment (sifnoc) [reply zk_prover/README.md]: Fixed at https://github.com/summa-dev/summa-solvency/pull/220/commits/298e83efc239e4dcd45c7b0794178b44695d8e5b

File: zk_prover/benches/utils.rs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+extern crate proc_macro;
```
[the remark this answers is not in the document]
Comment (sifnoc) [reply zk_prover/benches/utils.rs]: It's leftover, removed at https://github.com/summa-dev/summa-solvency/pull/220/commits/298e83efc239e4dcd45c7b0794178b44695d8e5b

**Comment by sifnoc** [comment]
Re open this PR as "Fixed benchmark for v1-improvement"

**Pull request closed without merging**